### ISSUE TITLE
[Chips] Provide feedback when a Chip is selected or deleted in a ChipField

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3190]
### Overview
In the "ChipsInputExampleViewController", there is no feedback when a user selects or deletes a Chip in a ChipField. The only sound heard is the one that is heard when there are no additional elements to navigate to.  Instead, some sort of feedback, like "Selected %@, backspace again to delete" might be appropriate.  Followed by "Deleted %@". If the chip loses selection, perhaps a similar notification?

Googlers, see internal document go/mdc-ios-chips-a11y-ux for details.

### Reproduction steps
1. Open the Catalog &rarr; Chips &rarr; Input
1. Use VoiceOver to enter, select, delete chips

**Expected Result**
1. Feedback via VoiceOver for Create, Select, Delete chips

**Actual Result**
1. Feedback only available when creating a Chip (it reads the Chip text).

### Version number
50.0.0

---
#### Internal data
- Associated internal bug: [b/117179108](http://b/117179108)

**Comment by jverkoey** [comment]
Specific repro steps:

1. Add multiple chips.
2. Start typing in order to create a new chip.
3. Start deleting text until a chip is selected.
4. Delete the chip.

Expected behavior between 3-4 is that some sort of VoiceOver feedback is given, but none presently is.

Expected behavior for step 4 is that some sort of VoiceOver feedback is given, but none presently is.

**Comment by jverkoey** [comment]
We may want to look into how this behavior should be implemented. May want to ping @dshoukry for clarification.

**Comment by romoore** [comment]
I have a meeting with Dahlia tomorrow.

**Comment by romoore** [comment]
The internal guidelines for Chips accessibility are still being worked out.

**Comment by material-mirror** [comment]
The internal issue [b/117179108](http://b/117179108) is now closed. This issue is being closed as a result.
